account-compression/programs/noop/Cargo.toml

```
@@ -16,4 +16,4 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-solana-program = "1.17.13"
+solana-program = ">=1.17.13,<=2.0.0"
```

I did a little test, and specifying it in this way means that it will only ever pull in "2.0.0" and nothing higher. Would it make more sense to relax this further as:

```suggestion
solana-program = ">=1.17.13,<=2"
```

Then it can eventually pick up v2.1

---

Yeah good idea. I was thinking it should be as restrictive as possible but we can always tighten the bound once we introduce an incompatibility.